charts/opentelemetry-collector/values.yaml

```diff
@@ -315,8 +315,9 @@ serviceMonitor:
 # PodDisruptionBudget is used only if standaloneCollector enabled
 podDisruptionBudget:
   enabled: false
+  # Use minAvailable or maxUnavailable option
   minAvailable: 2
-  maxUnavailable: 1
+  # maxUnavailable: 1
```

Comment:
You also need to wrap usage of these vars in a condition like 

```
{{- with podDisruptionBudget.minAvailable }}
minAvailable: {{ . }}
{{- end }}
``` 


Or, as another option, you can use the whole map as is:
```
{{ podDisruptionBudget | nindent 2 }}
```

but in this case `enabled` must be removed. Instead of `enabled` we can check if `podDisruptionBudget` has any fields defined in it. values.yaml would be the following by default:
```
podDisruptionBudget: {}
```
And PodDisruptionBudget template would start with
```
{{- if and .podDisruptionBudget standaloneCollector.enabled }}
```

Reply:
Where would this `{{ podDisruptionBudget | nindent 2 }}` be placed?

Reply:
in the template

Reply:
`{{ .Values.podDisruptionBudget | toYaml | trim | indent 2 }}` to be precise